### ISSUE TITLE
Add a setting to enable non-living entities in the stats

### ISSUE THREAD

**Issue by yoyo3841** [TheCSMods/mc-better-stats#36]
Include a setting to add another tab that only includes non-living entities such as boats, armour stands, etc..

I just think it would be funny

**Comment by TheCSDev** [comment]
Currently thinking about how handling non-living entities is gonna work out when it comes to the Issue #35 you submitted.

Right now I have to face a technical limitation in Java, and it has to do with generic types.
Long story short is that because of how Java handles generic types, it is impossible for me to check if an `EntityType<Entity>` is an `EntityType<LivingEntity>`. In other words, it is gonna be very hard for me to tell if an `Entity` is a `LivingEntity` prior to displaying it on the BSS, which I was supposed to use to filter out non-living entities out. That's why I chose to go with filtering out miscellaneous entities instead, without realizing I also filtered out villagers. I'll work something out tho, gotta take miscellaneous modded entities into account as well.

As for this feature, I am thinking of including non-living entities only if the player for some reason has stats on them. Like for example, if a player has a kill count on boats for some reason, I plan on showing that. Also fun fact (or not, idk), because non-living entities are not `LivingEntity`-ies, it is impossible to render them on the screen, which is why I coded them to render text of their names instead, like so:
<details>
<summary>What non-living entity stats look like (click here to expand)</summary>
<img src="https://user-images.githubusercontent.com/66475965/221346677-01cea1ae-58b9-4418-8683-abff0ad9040c.png"></img>
</details>
